### DISCUSSION TITLE
xcodebuild error: Kingfisher_Kingfisher does not support provisioning profiles. Kingfisher_Kingfisher does not support provisioning profiles, but provisioning profile XXX has been manually specified. Set the provisioning profile value to "Automatic" in the build settings editor. (in target 'Kingfisher_Kingfisher' from project 'Kingfisher')

### DISCUSSION THREAD

**Discussion by cuongchau93** [onevcat/Kingfisher#2215] [Q&A] [answered]
Hi all, 

On upgrading to Kingfisher 7.10.0, I start to receive the above error when trying to archive my project with manual signing

The command I'm using is 

> xcodebuild -project ./Abc.xcodeproj -scheme Scheme archive -sdk iphoneos -configuration AdHoc -archivePath ./archived CODE_SIGN_STYLE=Manual "CODE_SIGN_IDENTITY=XXX" PROVISIONING_PROFILE=XXX PROVISIONING_PROFILE_SPECIFIER=

I have looked through some online resources but couldn't find a solution 
Some related site about this build issue is 
https://forums.swift.org/t/xcode-14-beta-code-signing-issues-when-swiftpm-targets-include-resources/59685

I checked through the changes between 7.10 version and its older versions but couldn't find anything related. 

Any suggestion will be helpful.
Thanks for your help.

**Comment by onevcat** [comment]
@cuongchau93 Thanks for reporting this.

I am not sure what it can be now...

Did you mean a version before, such as [7.9.1](https://github.com/onevcat/Kingfisher/releases/tag/7.9.1), does not cause this error?

And may I know how did you add Kingfisher to your project? If you are trying to link Kingfisher.framework, maybe it is a good idea to set it as "Embed Without Signing" in your case.

![截屏2024-03-06 22 08 57](https://github.com/onevcat/Kingfisher/assets/1019875/71fc6b25-e486-4bfe-8a38-610537f5e71a)

**Reply by cuongchau93** [reply, marked as answer]
Thanks for your reply @onevcat 

I think my question is misleading
I later found out that if I upgraded my SPM to newer versions, it starts giving the same issue

The fix I found was to change the xcodebuild command a bit 
> xcodebuild -project ./Abc.xcodeproj -scheme Scheme archive -sdk iphoneos -configuration AdHoc -archivePath ./archived CODE_SIGNING_REQUIRED=Yes CODE_SIGNING_ALLOWED=Yes

With this it will help use default signing option of each package

**Comment by onevcat** [comment]
Thanks for the feedback! Nice to hear the solution!
